qokit/qaoa_circuit_sk.py

```diff
@@ -0,0 +1,139 @@
+###############################################################################
+# // SPDX-License-Identifier: Apache-2.0
+# // Copyright : JP Morgan Chase & Co
+###############################################################################
+# QAOA circuit for S_k
+
+import networkx as nx
+import numpy as np
+from qiskit import QuantumCircuit, QuantumRegister, ClassicalRegister
+from qiskit.circuit import ParameterVector
+from typing import Sequence
+
+
+def append_zz_term(qc, q1, q2, gamma):
```

Comment:
Appears to duplicate the function in qokit/qaoa_circuit_maxcut.py; idea: create qaoa_circuit_utils.py and import from there. Same for others below

Reply:
Done!